rend3-gltf/src/lib.rs

```diff
@@ -65,13 +65,29 @@ pub struct Mesh {
     pub primitives: Vec<MeshPrimitive>,
 }
 
+/// A set of [`SkeletonHandle`]s, one per mesh in the wrapping object, plus the
+/// index of the skin data in the `skins` array of [`LoadedGltfScene`].
+///
+/// All the skeletons are guaranteed to have the same number of joints and
+/// structure, but deform different meshes of the object.
+#[derive(Debug, Clone)]
+pub struct Armature {
+    /// The list of skeletons that are deforming this node's mesh primitives.
+    pub skeletons: Vec<SkeletonHandle>,
+    /// Index to the skin that contains the inverse bind matrices for the
+    /// skeletons in this armature.
+    pub skin_index: usize,
+}
+
```

Comment:
I chose `Armature` for lack of a better name. But essentially this is a struct that groups all of the `Skeleton`s in a single logical thing when there's a gltf node that has multiple skinned primitives.